src/content/configuration/experiments.mdx

```diff
@@ -35,6 +35,41 @@ module.exports = {
     topLevelAwait: true,
     asyncWebAssembly: true,
     layers: true,
+    lazyCompilation: true,
   },
 };
 ```
+
+### experiments.lazyCompilation
+
+Compile entrypoints and dynamic `import`s only when they are accessed.
+
+- Type
+  - `boolean`
+  - `object`
+    ```ts
+    {
+      // define a custom backend
+      backend?: ((
+        compiler: webpack.Compiler,
```

Comment:
Why in TS?

Reply:
It's almost impossible to document such a complex type without typescript.